Cargo.toml

```diff
@@ -13,6 +13,7 @@ keywords = ["encode", "decode", "serialize", "deserialize"]
 categories = ["encoding", "network-programming"]
 license = "MIT/Apache-2.0"
 edition = "2018"
+rust-version = "1.56.1"
```

Comment:
Is there a rationale for specifying this version?

Reply:
`1.56.1` is the current MSRV as determined using [cargo-msrv](https://github.com/foresterre/cargo-msrv)